x-pack/plugins/security_solution/public/management/pages/policy/view/event_filters/hooks.ts

```diff
@@ -66,6 +73,74 @@ export function useSearchAssignedEventFilters(
   );
 }
 
+export function useSearchNotAssignedEventFilters(
+  policyId: string,
+  options: { filter?: string; perPage?: number; enabled?: boolean }
+): QueryObserverResult<FoundExceptionListItemSchema, ServerApiError> {
+  const service = useGetEventFiltersService();
+  return useQuery<FoundExceptionListItemSchema, ServerApiError>(
+    ['eventFilters', 'notAssigned', policyId, options],
+    () => {
+      const { filter, perPage } = options;
+
+      return service.getList({
+        filter: parsePoliciesAndFilterToKql({
+          excludedPolicies: [policyId, 'all'],
+          kuery: parseQueryFilterToKQL(filter || '', SEARCHABLE_FIELDS),
+        }),
+        perPage,
+      });
+    },
+    {
+      refetchIntervalInBackground: false,
+      refetchOnWindowFocus: false,
+      refetchOnMount: true,
+      keepPreviousData: true,
+      enabled: options.enabled ?? true,
+    }
+  );
+}
+
+export function useBulkUpdateEventFilters(
```

Comment:
Generic bulk update action. Using this, we will be able to use the bulk API once is done without changing the components logic, only this hook

Reply:
nice!